src/renderer.js

```diff
@@ -0,0 +1,21 @@
+//const $ = require('jquery'); // eslint-disable-line no-unused-vars
+$(document).ready(function(){
+    function createPoint(e){
+        x = e.clientX;
+        y = e.clientY;
+        console.log('(' + x + ', ' + y + ')');
+        document.getElementById('dots').innerHTML += '<span class="point" style="left:' + e.clientX + 'px;top:' + e.clientY + 'px;"></span>'
+    }
+
+    var canvas = $('#canvas')
+    var ctx = canvas.getContext("2d");
+
+    $('#feild').mousemove(function(e){
+        x = e.clientX - 200;
+        y =  Math.round(window.innerHeight - e.clientY - (window.innerHeight/50)-2);
+        if(y<0){y=0}
+        $('#coords').text('(' + x + ', ' + y + ')');
+    });
+
+    $('body').click(createPoint);
```

Comment:
The goal here was (eventually) store the points, take a constant interval value, and generate points in real time. I don't know how memory intensive this will be, and we might want to implement a button to generate the path.